frontend/src/components/common/Resource/Resource.tsx

```diff
@@ -555,6 +562,63 @@ export function ContainerInfo(props: ContainerInfoProps) {
     );
   }
 
+  function handlePortForward(namespace?: string, podname?: string, containerPort?: number) {
+    if (!podname || !namespace || !containerPort) {
+      return;
+    }
+    window.desktopApi?.send('port-forward', { podname, namespace, targetPort: containerPort });
+  }
+
+  React.useEffect(() => {
+    window.desktopApi?.receive(
+      'port-forward-started',
+      ({ podname, namespace, cluster, port }: PortForwardElements) => {
+        if (
+          cluster === getCluster() &&
+          podname === props.podname &&
+          namespace === props.namespace
+        ) {
+          setIsCurrentPodPortForwarding(true);
+          setPortForwardServerURL(`http://localhost:${port}`);
+        }
+        setPortForwardStarted(true);
+      }
+    );
```

Comment:
If you listen to this in a higher component and store the info in redux, then you could just use the redux hook in this component to verify the states.

Reply:
this is now solved by using the backend api